### DISCUSSION TITLE
ANT BMS 7S-16S 80A supported?

### DISCUSSION THREAD

**Discussion by ferdixxxx** [syssi/esphome-ant-bms#79] [Q&A] [answered]
Hi,
i use the 7S-16S 80A and tried to use the esphome-ant-bms with home assistant.
Took me some time but now the esp is connected to home assistant.
But now matter which connection type i use MQttt or api no data is transfered.

As i don't know a solution i wanted to ask if it is the problem is that my 80A BMS is not supported or if there something else going wrong.

One thing is the 3V connection - as i don't have 3V yet i connected the d1 mini via USB.
Can this be the problem and i have to get a DC-DC Step down converter to get a decent voltage?

Thanks for your help and happy new year :-)

Ferdinand

**Comment by syssi** [comment]
Could you provide a photo of your wiring and the logbook of your ESPHome node?

**Comment by ferdixxxx** [comment]
Well - your question was hint enough...
Controlling the wiring before taking a photo brought up my mistake.
I connected the rx and tx wires to the corresponding pins at my wemos d1 mini. 
Just realized that this is not correct - you wiring scheme says GPIO4 and 5... (only saw the wiring with UART-TTL...)
So thank you very much!!

Would you better go with a step-down converter to get the demanded 3V or is my test version with a connected USB-Cable as power supply ok?

Thank you very much !!!

**Reply by syssi** [reply]
The Wemos D1 Mini is equipped with a dc-dc converter. There should be a 3.3V pin.

**Reply by ferdixxxx** [reply]
Yes, i know.
I am just a little bit concerned because the author of "https://github.com/KlausLi/Esp-Soyosource-Controller" says you should not use the USB connection as permanent power supply but only for testing... I just ordered some dc-dc converter and change my setup later on ...
I am asked because i was interested in your opinion :-)

**Reply by syssi** [reply]
I've no strong opinion here. :-) If the setup runs stable the power source is fine. ;-) If you try to build a low power setup avoiding the AMS and on-board LEDs is a good idea.

**Reply by syssi** [reply]
Are you able to talk to your BMS now? I would like to extend the list of supported devices and close this discussion some day.

**Comment by ferdixxxx** [comment, marked as answer]
Works like a charme - Thank you for all your efforts !!!

![Ant-BMS](https://github.com/syssi/esphome-ant-bms/assets/155317564/85a9b681-1a0c-4449-b717-afa92741dbb8)

**Comment by ferdixxxx** [comment]
Do you think its possible to connect my bms 3,8" display parallel to the esp or will there be a conflict/problem?

**Reply by syssi** [reply]
If he display is supported by ESPHome it should just work.

**Reply by ferdixxxx** [reply]
Ah, you are right... I'll get one that works with esp home. I am pretty sure the one delivered with the BMS is not suitable with esp home. 

Is there any chance to get the error messages delivered via your implementation? 

Is the button_ant_bms_restart corresponding to the reset button in the Bluetooth app? From time to time I need to reset the device to get rid of the error messages and get back in a properly running state. (The factory reset would be to much obviously... Don't want to start the complete configuration again)
